src/components/__tests__/__snapshots__/Selector-test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Selector should render <Selector/> with numbers 1`] = `
 <select
-  className="custom-root"
+  className={undefined}
```

Comment:
Should we update this test to not have undefined inside it? And check if select css class is actually read?